pkg/snapshotmgr/snapshot_manager.go

```diff
@@ -114,10 +115,24 @@ func NewSnapshotManagerFromCluster(params map[string]interface{}, config map[str
 	}
 	logger.Infof("SnapshotManager: Get ivdPETM from the params map, VirtualCenter=%v, port=%v", params["VirtualCenter"], params["port"])
 
+	// TODO: Initialize PVC PETM.
+
+	// Initialize dummy s3 config.
+	s3Config := astrolabe.S3Config{
+		Port:      0,
+		Host:      nil,
+		AccessKey: "",
+		Secret:    "",
+		Prefix:    "",
+		URLBase:   "VOID_URL",
+	}
+
+	// Initialize the DirectProtectedEntityManager
+	depm := server.NewDirectProtectedEntityManager([]astrolabe.ProtectedEntityTypeManager{ivdPETM}, s3Config)
 	snapMgr := SnapshotManager{
 		FieldLogger: logger,
 		config:      config,
-		ivdPETM:     ivdPETM,
+		pem:         depm,
```

Comment:
dpem?

Reply:
Dave suggested using pem, which represents the interface.